pkg/tcpip/BUILD

```diff
@@ -18,6 +18,8 @@ go_template_instance(
 go_library(
     name = "tcpip",
     srcs = [
+        "aligned.go",
+        "aligned_unsafe.go",
```

Comment:
nit: for clarity, it might be nice to name these something like `aligned_64bit.go`, `aligned_32bit_unsafe.go`.

Reply:
Done.